aws/resource_aws_route53_resolver_endpoint.go

```diff
@@ -78,7 +79,7 @@ func resourceAwsRoute53ResolverEndpoint() *schema.Resource {
 			"name": {
 				Type:         schema.TypeString,
 				Optional:     true,
-				ValidateFunc: validateRoute53ResolverEndpointName,
+				ValidateFunc: validateRoute53ResolverName,
```

Comment:
Since all Route 53 Resolver resource names have the same format, rename `validateRoute53ResolverEndpointName` to `validateRoute53ResolverName` and use in other Route 53 Resolver resources.